src/client/data/DatabasePouchDB.js

```diff
@@ -57,21 +62,18 @@ export const addUser = async (user) => {
     matches: user.matches
   };
 
-  return db.put(newUser);
+  return { user: db.put(newUser), id };
 };
```

Comment:
Returns new user's id so we can continue updating that user in the other CreateAccount views